ripe/atlas/tools/commands/configure.py

```diff
@@ -31,8 +30,8 @@ def add_arguments(self):
         self.parser.add_argument(
             "--init",
             action="store_true",
-            help="Create a configuration file and save it into your home directory at: {0}".format(
-                Configuration.USER_RC)
+            help="Create a configuration file and save it into your home "
+                 "directory at: {}".format(Configuration.USER_RC)
         )
```

Comment:
I think that Python has an odd way of handling multi-line strings.  It appears that if the string is within parentheses, then you don't need to tack `\` onto the end of every line because the `()` implies it.